services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -137,13 +145,19 @@ export const VerticleTable = (
             <Text variant="grey">Submission name</Text>
             <HStack>
               {showEditNameColumn && (
-                <button onClick={() => openAddEditReportModal(report)}>
-                  <Image
-                    src={editIcon}
-                    alt="Edit Report Name"
-                    minW={"1.75rem"}
-                  />
-                </button>
+                <>
+                  {report.status !== ReportStatus.SUBMITTED ? (
+                    <button onClick={() => openAddEditReportModal(report)}>
+                      <Image
+                        src={editIcon}
+                        alt="Edit Report Name"
+                        minW={"1.75rem"}
+                      />
+                    </button>
+                  ) : (
+                    <div />
+                  )}
```

Comment:
Same suggestion as above:

```suggestion
                  ) : null
                  }
```